ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -576,14 +687,108 @@
 				);
 				PRODUCT_BUNDLE_IDENTIFIER = eu.jhass.insporation;
 				PRODUCT_NAME = "$(TARGET_NAME)";
-				PROVISIONING_PROFILE_SPECIFIER = "match AppStore eu.jhass.insporation";
+				PROVISIONING_PROFILE_SPECIFIER = "";
```

Comment:
👀

Reply:
Same here. Xcode managed seems to be the better solution for this "Profiling" section.